### ISSUE TITLE
Travis CI needs to recognize unittest failures

### ISSUE THREAD

**Issue by bytesized** [mozilla/Snappy-Symbolication-Server#61]
It seems that Travis CI does not recognize unittest test failures. See [this test run](https://travis-ci.org/mozilla/Snappy-Symbolication-Server/builds/170567081) as an example.

**Comment by willkg** [comment]
Travis has this at the bottom of the log:

```
The command "python ./runTests.py -c ./travis_config.json" exited with 0.
```

It `runTests.py` needs to return a non-0 exit code if tests fail.

**Comment by bytesized** [comment]
I thought it did. Obviously I was wrong. I will investigate.

**Comment by bytesized** [comment]
Fixed by PR #62
